src/main/java/com/labwithcommunity/domain/user/UserEntity.java

```diff
@@ -1,4 +1,35 @@
 package com.labwithcommunity.domain.user;
 
-public class UserEntity {
+import jakarta.persistence.*;
+import lombok.*;
+
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode
+@Table(name = "Users")
+class UserEntity {
```

Comment:
You might want to consider using the @Data annotation instead of @Getter, @Setter, etc.